packages/legacy/core/__tests__/components/__snapshots__/CameraDisclosureModal.test.tsx.snap

```diff
@@ -99,7 +99,7 @@ exports[`CameraDisclosureModal Component Renders correctly 1`] = `
         }
       >
         <View
-          accessibilityLabel="CameraDisclosure.Allow"
+          accessibilityLabel="Global.Continue"
```

Comment:
should we remove `CameraDisclosure.Allow` from the index files

Reply:
Done